reports/Kentucky.md

```diff
@@ -119,3 +119,15 @@ id: ky-louisville-7
 **Links**
 
 * [Video taken from across the street](https://www.reddit.com/r/Louisville/comments/gw5joa/peaceful_protesters_marching_in_st_mathews_get/)
+
+### Police dogpiling a protestor and then targets a witness | June 16th
+
+A guy records a half-dozen policemen dogpiling a protestor and then one of them shoots the window to stop him from recording.
+
+tags: rubber-bullets, shoot
```

Comment:
Can you update the tags to include attacking the protestor? I would suggest `tackle, beat, baton`

Reply:
I also added these tags.